### PR TITLE
My Home: Add new eCommerce cards to Learn & Grow section

### LINKED ISSUE #54863: My Home: eCommerce Learn & Grow cards
## What
Add a new card to the Learn & Grow section of My Home that is tailored to eCommerce users.

This is an example of a Learn & Grow card:
image
 
## Design
@jeffgolenski 
- [x] work with @jordesign & Kate (I couldn't find Kate's github handle) to pick [best candidate article(s) from Seth's list](https://docs.google.com/spreadsheets/d/1mI6Wdo1H0zDu7dV6xlqvSi2uUfkU-zv7OfDU3xwSGa4/edit#gid=0) for a new Learn & Grow Card 
- [x] Design Card: copy & visuals

## Dev
- [x] add these two cards to the Learn & Grow section #55014
- [x] filter this card(s) visibility so it only shows to eCommerce users D64875-code

## Testing

### PR DESCRIPTION
**Pull request by arthur791004**
#### Changes proposed in this Pull Request

* Add new cards under `Learn & Grow` section for the user under the eCommerce plan
   * 10 ways to find success with your website
   * How to respond to customer feedback

image
image

https://user-images.githubusercontent.com/13596067/127460237-c5151ddb-ce35-46e7-86a5-19188a74b228.mov

#### Testing instructions

* Apply D64875-code
* Go to My Home
* Switch cards in `Learn & Grow` section
   * You have to see the new cards if the site is under the eCommerce plan
   * You should not see the new cards if the site is under the eCommerce plan

Close https://github.com/Automattic/wp-calypso/issues/54863

### CODE REVIEW

File: client/my-sites/customer-home/cards/education/find-success/index.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import EducationalContent from '../educational-content';
+
+const FindSuccess = () => {
+	const translate = useTranslate();
+
+	return (
+		<EducationalContent
+			title={ translate( '10 ways to find success with your website' ) }
+			description={ translate(
+				"You need to keep traffic coming to your site so you get leads and sales. We'll teach you know"
```
Comment (Aurorum) [inline client/my-sites/customer-home/cards/education/find-success/index.jsx]: I think there's a typo here!

```suggestion
				"You need to keep traffic coming to your site so you get leads and sales. We'll teach you how."
```

**Review by Aurorum — CHANGES_REQUESTED** [review]

File: client/my-sites/customer-home/cards/education/find-success/index.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+			) }
+			links={ [
+				{
+					externalLink: true,
+					url:
+						'https://wordpress.com/go/tips-and-tricks/10-ways-to-find-success-with-your-new-small-business-website/',
```
Comment (p-jackson) [inline client/my-sites/customer-home/cards/education/find-success/index.jsx:21]: We have a function called `localizeUrl()` which attempts to change the url for a support site link to one of our non-en support sites. I'm not sure we have translated versions of the `/go` site, but we should use `localizeUrl()` for these links as long as it doesn't break anything. That way if the `/go` site ever was translated these links would go the correct place.

Comment (arthur791004) [reply client/my-sites/customer-home/cards/education/find-success/index.jsx:21]: I test the url with `localizeUrl` earlier, but I'm still not sure we have translated versions or not. So I remove that function here 😅

Comment (p-jackson) [reply client/my-sites/customer-home/cards/education/find-success/index.jsx:21]: Unfortunately these articles don't exist on the translated version of `/go` e.g. `https://wordpress.com/es/go`

Comment (p-jackson) [reply client/my-sites/customer-home/cards/education/find-success/index.jsx:21]: > I test the url with `localizeUrl` earlier, but I'm still not sure we have translated versions or not. So I remove that function here 😅

I missed this comment! Nice job, yeah usually that function should work and I'm going to let the team know that we're now linking to English only articles.

**Review by p-jackson — APPROVED** [review]

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6303159

Thank you @arthur791004 for including a screenshot in the description! This is really helpful for our translators.

**Comment by autumnfjeld** [comment]
Nice work :)

eCommerce Site:
grow-and-learn-ecommerce-Screen Capture on 2021-07-30 at 16-09-05

non-eCommerce Site:

image

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6303159).

**Pull request merged**